model/src/application/interaction/application_command_autocomplete/option.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
+use serde_repr::{Deserialize_repr, Serialize_repr};
+
+/// Data received when a user partially fills in a command option.
+///
+/// [`value`]: Self::value
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct ApplicationCommandAutocompleteDataOption {
+    #[serde(default)]
+    pub focused: bool,
+    #[serde(rename = "type")]
+    pub kind: ApplicationCommandAutocompleteDataOptionType,
+    pub name: String,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub options: Vec<ApplicationCommandAutocompleteDataOption>,
+    pub value: String,
```

Comment:
```suggestion
    #[serde(default)]
    pub value: Option<String>,
```
To account for subcommands